packages/node_modules/cerebral/src/test/index.test.js

```diff
@@ -53,114 +53,6 @@ describe('test helpers', () => {
     })
   })
 
-  describe('runSignal', () => {
```

Comment:
I think the test for `runSignal` should not be removed. Only for `RunSignal factory`.

Reply:
Ok I have fixed that.